javascript/script.js

```diff
@@ -0,0 +1,8 @@
+// .script
+
+var withButtons = document.getElementsByClassName('button');
+	console.log(withButtons);
+	for (var i = 0; i < withButtons.length ; i++) {
+		alert(withButtons);
+		document.getElementsByClassName('button').innerHTML = withButtons;
```

Comment:
`withButtons` to tablica. `innerHTML` na tablicy nie zadziała, natomiast na jej pojedynczych elementach już tak. Do elementów tablicy dostajemy się za pomocą indeksu, np:
```
var tablica = ['Ala', 'Kot', 'Pies'];
console.log(tablica[2]); // 'Pies'
```